packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.js

```diff
@@ -22,14 +22,12 @@ const propTypes = {
   trademark: PropTypes.string,
   /** The URL of the image for the brand */
   brandImageSrc: PropTypes.string.isRequired,
+  /** The URL of the image for the background */
+  bgImgSrc: PropTypes.string,
   /** The alternate text of the brand image */
-  brandImageAlt: PropTypes.string.isRequired,
-  /** The URL of the image for the logo */
-  logoImageSrc: PropTypes.string,
-  /** The alternate text of the logo image */
-  logoImageAlt: props => {
-    if (props.logoImageSrc && !props.logoImageAlt) {
-      return new Error('logoImageAlt is required when a logoImageSrc is specified');
+  brandImageAlt: props => {
```

Comment:
since brandImageSrc is already required, we can also simply set brandImageAlt to required.
Although i'm questioning whether the brand image should be required...

Reply:
For now I left it as required, can change at a later point if you like. The main focus for this PR is just to make the background image configurable.